spec/models/item_season_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 RSpec.describe ItemSeason, type: :model do
   let!(:produce_item) { create(:produce_item) }
-  let!(:item_season) { described_class.new(produce_item_id: produce_item.id, month_index: 1, country_code: 'br') }
+  let!(:item_season) { described_class.new(produce_item_id: produce_item.id, month_index: 1, country_code: 'BR') }
```

Comment:
**suggestion:** this could become `subject!(:item_season)`

Reply:
**suggestion:** remove `!` from variables. I don't think we need these here.

Reply:
So I read a bit on subject vs. let, and it seems like it makes sense here because we are only testing for some active records validations with shoulda matchers. It is pretty clear what the subject is, and what is being tested. Is this correct?

Reply:
Yeap, we should always mark what we're testing with `subject` :D